src/domUpdates.js

```diff
@@ -204,6 +214,32 @@ const updateCurrentRecipe = recipeCard => {
   pageData.currentRecipeCard = getRecipeCard(thisRecipe);
 }
 
+const populateRecipeName = currentRecipe => {
+  document.querySelector('#recipeName').innerHTML = `  <h1>${currentRecipe.name}</h1>  `
+}
+
+const populateAddBtn = () => {
+  // const recipeSaved = user.recipesToCook.some(recipe => recipe.id === currentRecipe.id);
+  // let buttonText;
+  // let buttonStatus;
+  // if (!recipeSaved) { 
+  //   buttonText = "Add to Save to Cook Recipes";
+  //   buttonStatus = "add-recipe modal-recipe-btn";
+  // } else {
+  //   buttonText = "Remove from Save to Cook Recipes";
+  //   buttonStatus = "remove-recipe modal-recipe-btn";
+  // };
+//saving to use once currentUser is accessible
+  const buttonStatus = "add-recipe modal-recipe-btn";
+  const buttonText = "Add to Saved Recipes";
+  document.querySelector('.modal-add-recipe').innerHTML = 
+  `
```

Comment:
If this is eventually going to toggle back and forth depending on whether they can add or remove recipes, should the class of the section be something more generic like 'modal-bookmark-button'  instead of 'modal-add-recipe'  since eventually it's task will not be to `add` but to `remove`?

Reply:
Hey Laura. Good catch. So I initially wanted to implement the code that you see is commented out. 
That way, we can use the same function for both adding and removing. 
We just need to ensure that when an add button is clicked, we change its class from "add-recipe" to "remove-recipe" if that makes sense.